l10n_it_ricevute_bancarie/models/account/account.py

```diff
@@ -170,9 +170,11 @@ def month_check(self, invoice_date_due, all_date_due):
         :param all_date_due: list of due dates for partner
         :return: True if month of invoice_date_due is in a list of all_date_due
         """
-        for d in all_date_due:
-            if invoice_date_due[:7] == str(d.strftime('%Y-%m')):
-                return True
+        self.ensure_one()
+        if self.partner_id.riba_policy_expenses in ['one_a_month', False]:
```

Comment:
Sostituiresti la lista con una tupla, per favore? Aggiungerei la stringa vuota anche

`if self.partner_id.riba_policy_expenses in ('one_a_month', '',  False):`

Reply:
👍